### ISSUE TITLE
Consider reaching out to redcap to ask for test server

### ISSUE THREAD

**Issue by pwildenhain** [redcap-tools/PyCap#131]
This reminds me of a setup I read about in the [`jira`](https://jira.readthedocs.io/en/master/) python package, where Atlassian provided the team with a test server. This would be really nice, at least for running in CI.

**Comment by remifrazierucsf** [comment]
Is this still an issue @pwildenhain ?

**Comment by pwildenhain** [comment]
Yup, though I haven't reached out to REDCap yet to ask. Do you want to reach out?
